net/pfSense-pkg-cellular/Makefile

```diff
@@ -1,7 +1,7 @@
 # $FreeBSD$
 
 PORTNAME=	pfSense-pkg-cellular
-PORTVERSION=	1.2.1
+PORTVERSION=	1.2.2
 PORTREVISION=	1
```

Comment:
When PORTVERSION changes, PORTREVISION line must be deleted

Reply:
I removed the PORTREVISION